mappings/net/minecraft/container/Container.mapping

```diff
@@ -22,6 +22,7 @@ CLASS att net/minecraft/container/Container
 		ARG 1 slotId
 		ARG 2 clickData
 		ARG 3 actionType
+	METHOD a quickCraftShouldContinue (ILary;)Z
```

Comment:
shouldQuickCraftContinue? shouldMassCraftContinue? What's the Mojang term for this?

Reply:
the method is related to the SlotActionType.QUICK_CRAFT, and I wrote the name in that order so that it groups up with other methods/fields related to quick crafting, like `quickCraftEnd()`, `quickCraftButton`, `quickCraftSlots`, etc...

Reply:
We don't forcibly prefix, as per convention, though?

Reply:
sounds fair, renamed quickCraftShouldContinue -> shouldQuickCraftContinue and quickCraftEnd -> endQuickCraft.